Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [workspace]
 members = [
-    "import_streets",
-    "osm2streets",
+    "streets_reader",
```

Comment:
`streetsio`?

Reply:
Maybe? Or `osm_reader` to emphasize the source, but maybe that just repeats the main crate name? Not opinionated here

Reply:
I like `streetsio` as an analogy to [osmio](https://crates.io/crates/osmio), but I'll leave it as is until its context in the wider ecosystem becomes more obvious.